dstar_main:
  ros__parameters:
    esdf_or_grid_map: true # false: esdf, true: grid_map
    read_in_topic: /global_costmap/costmap
    path_topic: /dstar_plan
    goal_topic: /clicked_point
    robot_frame: base_link
    map_frame: map